### PR TITLE
CLDR-16169 Reset namePattern and nameField to draft="provisional"

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-16169

- [ ] This PR completes the ticket.

### CODE REVIEW

File: common/main/en.xml [hunk 1]
```diff
@@ -9305,130 +9305,130 @@ annotations.
 		<initialPattern type="initial">{0}.</initialPattern>
 		<initialPattern type="initialSequence">{0} {1}</initialPattern>
 		<personName order="givenFirst" length="long" usage="referring" formality="formal">
-			<namePattern>{title} {given} {given2} {surname} {generation}, {credentials}</namePattern>
+			<namePattern draft="provisional">{title} {given} {given2} {surname} {generation}, {credentials}</namePattern>
 		</personName>
```
Comment (pedberg-icu) [inline common/main/en.xml:9309]: We don't want to reset _`en`_ to provisional, do we??

Comment (macchiati) [reply common/main/en.xml]: Right, realized that I'd not excluded en.xml part-way through the meeting!

Comment (macchiati) [reply common/main/en.xml]: Fixed that and a couple of other issues.

**Review by AEApple — APPROVED** [review]
LGTM, but I only spot checked

**Comment by macchiati** [comment]
Thanks! With this many files, spot-checking is exactly the right approach. (I think the tests would catch anything serious.)

**Pull request merged**